packages/DataGrid/src/components/Cell/Cell.js

```diff
@@ -19,10 +19,12 @@ const propTypes = {
   style: PropTypes.shape({}).isRequired,
   /** Add an alternate background on the DataGrid's rows */
   hasZebraStripes: PropTypes.bool.isRequired,
+  /** Define the look for borders in the table */
+  borderType: PropTypes.oneOf(["grid", "empty", "horizontal", "vertical"]).isRequired,
```

Comment:
Use same constants mentioned in other comment

Reply:
👍